unfetter-discover/unfetter
# Issue 1119

I removed all the "links" in the ansible playbook roles, since Docker handles the DNS automatically through the Docker defined network

TO TEST
```
ansible-playbook deploy-dev.yml
```

The stack should start up without link errors
